pkg/apis/networking/validation/validation.go

```diff
@@ -123,6 +123,17 @@ func ValidateNetworkPolicySpec(spec *networking.NetworkPolicySpec, fldPath *fiel
 			}
 		}
 	}
+	// Validate PolicyTypes
+	if len(spec.PolicyTypes) > 2 {
```

Comment:
len 0 is also an error.
Also, this code would allow [ "Ingress", "Ingress" ]...